documentation/src/main/asciidoc/chapter-2-set-up.asciidoc

```diff
@@ -248,13 +248,11 @@ If a policy is given for a specific mapper via `@Mapper#unmappedTargetPolicy()`,
 |`WARN`
 |===
 
-=== Using MapStruct on Java 9
+=== Using MapStruct on Java 9 and higher versions
 
-MapStruct can be used with Java 9 (JPMS), support for it is experimental.
+MapStruct can be used with Java 9 and higher versions.
 
-A core theme of Java 9 is the modularization of the JDK. One effect of this is that a specific module needs	to be enabled for a project in order to use the `javax.annotation.Generated` annotation. `@Generated` is added by MapStruct to generated mapper classes to tag them as generated code, stating the date of generation, the generator version etc.
-
-To allow usage of the `@Generated` annotation the module _java.xml.ws.annotation_ must be enabled. When using Maven, this can be done like this:
+To allow usage of the `@Generated` annotation the module _java.xml.ws.annotation_ can be enabled. When using Maven, this can be done like this:
```

Comment:
The `javax.annotation.processing.Generated` annotation should be mentioned instead (from the jdk.compiler) module.

Reply:
5d09733